src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -477,6 +503,46 @@ impl LeaderBlockCommitOp {
                 return Err(op_error::BlockCommitBadOutputs);
             }
         };
+        Ok(())
+    }
+
+    fn check_after_pox_sunset(&self) -> Result<(), op_error> {
+        if self.commit_outs.len() != 1 {
```

Comment:
`commit_outs` only contains either PoX transfers or burns, right?  i.e. this check works even if there are change outputs (or even multiple change outputs)?

Reply:
Ah, I see you test this below.  Nevermind :)